.pre-commit-config.yaml

```
@@ -218,39 +218,26 @@ repos:
     - tests/
     pass_filenames: false
 
-- repo: local
+- repo: https://github.com/PyCQA/pylint
+  rev: v2.12.2
   hooks:
   - id: pylint
-    language: system
-    name: PyLint
-    files: \.py$
-    entry: python -m pylint
-    args:
-    - --ignore=tm_tokenize
-    - |-
-      --disable=
-      consider-using-with,
-      implicit-str-concat,
-      import-error,
-      invalid-name,
-      missing-class-docstring,
-      missing-function-docstring,
-      missing-module-docstring,
-      no-else-return,
-      no-member,
-      no-self-use,
-      protected-access,
-      redefined-builtin,
-      too-few-public-methods,
-      too-many-arguments,
-      too-many-branches,
-      too-many-instance-attributes,
-      too-many-locals,
-      too-many-statements,
-      unused-import,
-      useless-else-on-loop,
-      wrong-import-order,
-    stages:
-    - manual
+    additional_dependencies:
+      - ansible-core
+      - ansible-runner
+      - dataclasses; python_version<"3.7"
+      - libtmux
+      - onigurumacffi
+      - pytest
+      - pyyaml
+      - sphinx
+      - setuptools-scm
```

Let's be consistent about the indentation and unnecessary spaces + comments are needed to let people know where to copy-paste things from as a part of maintenance:
```suggestion
    # NOTE: The entries below must be kept in sync with the contents of `setup.cfg` at all times,
    # NOTE: including `install_requires` and any extras that may be imported in runtime to accommodate
    # NOTE: for `pylint`'s ability to import the modules being checked. They must also include any of the
    # NOTE: test and platform-dependent requirements that may be imported under different circumstances.
    - ansible-core  # dependency imported in utils/catalog_collections.py
    # setup.cfg:
    - ansible-runner ~= 2.0
    - dataclasses; python_version < "3.7"
    - importlib-resources; python_version < "3.9.0"
    - jinja2
    - onigurumacffi
    - pyyaml
    # tests:
    - libtmux
    - pytest
    # docs:
    - sphinx
    - setuptools-scm
```